x/gov/README.md

```diff
@@ -1149,10 +1149,14 @@ where `proposal.json` contains:
     }
   ],
   "metadata": "AQ==",
-  "deposit": "10stake"
+  "deposit": "10stake",
+  "title": "Proposal Title",
+  "summary": "Proposal Summary"
 }
 ```
 
+> Note: By default the metadata and the proposal are both limited by 255 characters, this can be overridden by the application developer
```

Comment:
```suggestion
:::note
By default the metadata and the proposal are both limited by 255 characters, this can be overridden by the application developer.
:::
```